docs/development/adding_labels_to_ces_resources_de.md

```diff
@@ -0,0 +1,163 @@
+
+---
+
+Labels sind Kubernetes-eigene Schlüssel-Wert-Paare, die an Kubernetes-Objekte angehängt werden, z. B. Deployments, Pods und so weiter. Sie haben nur für die Benutzer eine Bedeutung. Labels helfen Benutzern, Ressourcen zu identifizieren und zu organisieren, insbesondere mithilfe von Selektoren.
+
+Jeder Label-Schlüssel muss für ein bestimmtes Objekt eindeutig sein. Wird für denselben Schlüssel ein anderer Label-Wert festgelegt, wird der Wert überschrieben.
+
+Weitere Informationen über Labels und Selektoren finden Sie in der offiziellen [Kubernetes-Dokumentation] (https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/).
+
+## Label-Richtlinie für CES-Komponenten
+
+Die Label-Policy besteht im Wesentlichen aus zwei Regeln:
+
+1. zum Labeln einer bestimmten Anwendung sowie
+2. für die Kennzeichnung aller beweglichen Teile, nachdem die Maschine provisioniert wurde.
+
+Außerdem gibt es zwei weitere reservierte Labels, die automatisch von `k8s-dogu-operator` erzeugt werden:
+- `dogu.name: <dogu-name>` und
+- `dogu.version: <dogu-version`
+
+Diese Bezeichnungen **MÜSSEN** NICHT manuell gesetzt werden, da diese Bezeichnungen für die dogu-Upgrade-Prozedur entscheidend sind.
+
+### 1. Labels, die die Anwendung identifizieren
+
+**ALL** Ressourcen einer CES-Komponente **MÜSSEN** ein Label erhalten, das die Auswahl der zugrunde liegenden Anwendung ermöglicht. Damit können alle Ressourcen identifiziert werden, die durch den Einsatz dieser Anwendung in diesen Cluster gelangt sind.
+
+Das Format des Labels ist: `app.kubernetes.io/name: <component-name>`
+
+Ein Beispiel ist das vollständige Löschen einer Komponente, um sie durch eine andere Version zu ersetzen, die möglicherweise weniger oder andere Ressourcen mitbringt. Ohne eine ordnungsgemäße Kennzeichnung würde unbenutze Ressourcen hinterlassen, deren Nutzungsstatus unklar sein könnte.
+
+Diese Syntax ist den Helm-Anwendungskennzeichnungen entnommen.
+
+Beispiel:
+
+```yaml
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  labels:
+    app.kubernetes.io/name: your-ces-k8s-component-name
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    app.kubernetes.io/name: your-ces-k8s-component-name
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: your-ces-k8s-component-name
+```
+
+### 2. Labels, die als abnehmbarer Teil des Cloudogu EcoSystems identifizieren
+
+**Einige** Ressourcen einer CES-Komponente **MÜSSEN** ein Label erhalten, das alle gruppiert die Auswahl der zugrundeliegenden Anwendung ermöglicht. Dies ermöglicht es, alle Ressourcen zu identifizieren, die mit offiziellen Mitteln des Cloudogu EcoSystems in diesen Cluster gelangt sind.
```

Comment:
**Einige** Ressourcen einer CES-Komponente **MÜSSEN** ein Label erhalten, das alle gruppiert und die Auswahl der zugrundeliegenden Anwendung ermöglicht. Dies ermöglicht es, alle Ressourcen zu identifizieren, die mit offiziellen Mitteln des Cloudogu EcoSystems in diesen Cluster gelangt sind.